embulk-output-redshift/src/main/java/org/embulk/output/redshift/RedshiftCopyBatchInsert.java

```diff
@@ -94,6 +97,20 @@ public RedshiftCopyBatchInsert(JdbcOutputConnector connector,
                     + " IAM user needs \"s3:GetBucketLocation\" permission if Redshift region and S3 region are different.");
         }
         this.s3RegionName = s3RegionName;
+
+        if (copyIamRoleName != null && copyIamRoleName.length() > 0) {
+            String accountId = null;
+            if (copyAwsAccountId != null && copyAwsAccountId.length() > 0) {
```

Comment:
I think the condition `...length() > 0` is not necessary because "" is not the default value but the value user explicitly specified.
(an exception will be thrown in the AWS library)

Reply:
I consider safety, it is no necessary in this part.
And the behavior  when passing no null value(include "") depends on aws's specification and user expectation, I think too.
I'll fix it.